example/run_demo.py

```diff
@@ -216,6 +215,7 @@ def run_demo(report: Report, server: JsonRpcServer) -> None:
         for count, (file_path, incidents) in enumerate(impacted_files.items(), 1):
             for incident in incidents:
                 incident.uri = os.path.join(SAMPLE_APP_DIR, file_path)
+                incident.uri = Path(incident.uri).absolute().__str__()
```

Comment:
See below for comment about `.absolute()`.

Reply:
updated, see above comment.